### ISSUE TITLE
Validate instrument model in sample sheet and run outputs

### ISSUE THREAD

**Issue by ElDeveloper** [biocore/mg-scripts#4]
As part of the pipeline, it would be beneficial to check that the instrument model in the sample sheet matches the instrument code in the run's folder name. If this is not the case, then we should raise an error as it could imply barcodes in the wrong orientation, or a different problem.

**Comment by charles-cowart** [comment]
I don't see an instrument model in the current sample-sheets, and I'm not sure which part of a run_directory's format (e.g. 211021_A00000_0000_SAMPLE)  is the instrument code. I'm going to add MacKenzie to this project and confirm with her whether this is needed or not.

**Comment by charles-cowart** [comment]
@mmbryant23 Do you have any thoughts on the SPP having such a check?

**Comment by mmbryant23** [comment]
@charles-cowart I don't think this has been a pressing issue, especially because if barcodes were in the incorrect orientation, it would be very obvious post-processing (and this would be one of the first things to check for troubleshooting). Additionally, the iSeq/NovaSeq samplesheet generation steps have been further optimized in the notebook since this issue was originally opened, overall minimizing potential user error for generating the wrong type of samplesheet. If it was a straightforward check it wouldn't be a harmful check to have, but I think trying to figure out how to make this happen might take more effort than what it is worth.
